src/CustomElementInternals.js

```diff
@@ -208,6 +222,11 @@ export default class CustomElementInternals {
             this.patchAndUpgradeTree(importNode, visitedImports);
           });
         }
+      } else if (element.hasAttribute('is') || element.is) {
+        element.__CE_is = element.getAttribute('is') || element.is;
+        element.removeAttribute('is');
```

Comment:
We remove the is attribute so that browsers don't try to do anything native with it.